app/ide-desktop/lib/content/esbuild-config.ts

```diff
@@ -122,6 +123,16 @@ export function bundlerOptions(args: Arguments): esbuild.BuildOptions {
             // We do not control naming of third-party options.
             esbuildPluginAlias({ ensogl_app: ensoglAppPath }),
             esbuildPluginTime(),
+            // This must run before the copy plugin so that the generated `tailwind.css` is used.
+            {
+                name: 'enso-generate-tailwind',
+                setup: build => {
+                    build.onStart(() => {
+                        console.log(`Generating tailwind css from ${path.resolve(THIS_PATH, 'src', 'tailwind.css')} to ${path.join(outputPath, 'tailwind.css')}`)
+                        childProcess.execSync(`npx tailwindcss -i ${path.resolve(THIS_PATH, 'src', 'tailwind.css')} -o ${path.join(outputPath, 'tailwind.css')} -c ${path.resolve(THIS_PATH, 'tailwind.config.ts')} --minify`);
```

Comment:
Please avoid using the `exec*('command')` form whenever possible. Always prefer `exec*('program', [args])`. Otherwise, it is nearly impossible to support things like spaces in paths in a portable manner.

Also, please extract long, repetitive expressions like `path.resolve(THIS_PATH, 'src', 'tailwind.css')` to named variables. That will improve code clarity.

Reply:
changed to `spawnSync(program, args)`; extracted to named variables

Reply:
> changed to `spawnSync(program, args)`; extracted to named variables

Please double check that this process is killed when parent process is killed. We had issues with spawn in the past. CC @mwu-tow

Reply:
unfortunately this does seem to be a problem - i've tested by switching from `spawnSync` to `spawn`, adding a `process.exit(1)`, and checking if the child process is still running. i *think* avoiding `npx` fixes it, but the current solution i've ended up with is quite hacky